server/survey/collectImport/jobs/surveyCreatorJob.js

```diff
@@ -0,0 +1,37 @@
+const R = require('ramda')
+
+const Survey = require('../../../../common/survey/survey')
+const SurveyInfo = require('../../../../common/survey/_internal/surveyInfo')
```

Comment:
this is internal to survey, it cannot be exported here. export the SurveyInfo.keys from Survey as infoKeys

Reply:
rightissimo!